### PR TITLE
Revert 7760 fix ci pqueue

### PR DESCRIPTION
**Pull request by juliensnz**
**Description (for Contributor and Core Developer)**

We now transpile p-queue to avoid errors with firefox. I tested the compilation time and it's pretty much the same.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -160,8 +160,7 @@ const webpackConfig = {
       // Process the pim webpack files with babel
       {
         test: /\.js$/,
-        include: /(public\/bundles|webpack|spec)/,
-        exclude: /lib|node_modules/,
+        include: /(public\/bundles|webpack|spec|node_modules\/p\-queue)/,
```
Comment (tseho) [inline webpack.config.js]: `include` can be an array, IMHO, I think it's the perfect time to make it more readable.

Comment (tseho) [reply webpack.config.js]: https://webpack.js.org/configuration/module/#condition

Comment (juliensnz) [reply webpack.config.js]: I do'nt see the documentation for that

**Comment by tseho** [comment]
I did a diff before and after this commit and `main.min.js` shouldn't change but it does.
I found that removing `exclude: /lib|node_modules/,` does affect the result.
The output has light differences, eg:
```
< 
<     widget: function() {
---
>     widget: function widget() {
```

I don't know if we should care about these.

**Comment by juliensnz** [comment]
@tseho for me as long as the CI pass, it's OK :)

**Pull request merged**